### ISSUE TITLE
[blue-hyperdrive10] TypeError: Cannot read property 'key' of undefined {"category":"browser"}

### ISSUE THREAD

**Issue by almereyda** [beakerbrowser/beaker#1538]
Operation System: Fedora 30, with latest Node 10/LTS (10.20.1) and 12/LTS (12.16.1) via nvm.sh

Beaker version: 941031d8bd314ebab1c935fea36c0995fa3bb234

```
# rm -rf ~/.config/Beaker\ Browser
# rm -rf node_modulse app/node_modules
# nvm install 10.20.1 # or # 12.16.1
# npm install
# npm rebuild
# npm start

...

Unhandled Rejection at: Promise Promise {
  <rejected> TypeError: Cannot read property 'key' of undefined
      at setup$l (/home/yala/src/github.com/beakerbrowser/beaker/app/main.build.js:10010:48)
      at async App.<anonymous> (/home/yala/src/github.com/beakerbrowser/beaker/app/main.build.js:15803:3)
} reason: TypeError: Cannot read property 'key' of undefined
    at setup$l (/home/yala/src/github.com/beakerbrowser/beaker/app/main.build.js:10010:48)
    at async App.<anonymous> (/home/yala/src/github.com/beakerbrowser/beaker/app/main.build.js:15803:3)
error:    Unhandled Rejection at: Promise [object Promise] reason: TypeError: Cannot read property 'key' of undefined {"category":"browser"}
Hyperdrive daemon has been restored
```

and then no Electron user interface appears, but the application remains running and reports things such as

    info:     Running GC {"category":"filesystem","subcategory":"trash-collector"}

**Comment by pfrazee** [comment]
My bad, looks like I broke the code-path for users on initial load. Fixed with 2ccb2d82a85188320fb75a89437802406802a208.

**Comment by almereyda** [comment]
Yes, `blue-hyperdrive10` now builds and runs.
